extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -149,6 +149,14 @@ core::Property ConsumeWindowsEventLog::BookmarkRootDirectory(
   withDescription("Directory which contains processor state data.")->
   build());
 
+core::Property ConsumeWindowsEventLog::ProcessOldEvents(
+  core::PropertyBuilder::createProperty("Process Old Events")->
+  isRequired(true)->
+  withAllowableValues<std::string>({ "true", "false" })->

```

Comment:
In case you add a bool validator, you don't need this, but it becomes more consistent with C2.

Reply:
Fixed.